### PR TITLE
[#223759] - Callback

### PR DESCRIPTION
**Pull request by HanThiKimNgoc**
https://dev.framgia.com/issues/223747
https://dev.framgia.com/issues/223748
https://dev.framgia.com/issues/223749
https://dev.framgia.com/issues/223750
https://dev.framgia.com/issues/223752
https://dev.framgia.com/issues/223754
https://dev.framgia.com/issues/223755
https://dev.framgia.com/issues/223756
https://dev.framgia.com/issues/223757
https://dev.framgia.com/issues/223759
https://dev.framgia.com/issues/223760
https://dev.framgia.com/issues/223761

### CODE REVIEW

File: callback/client.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	Key string
+}
+
+const (
+	baseURL = "api/public/v3/media/entity/callback"
+)
+
+func getC() Client {
+	b := uiza.GetBackend(uiza.APIBackend)
+	b.SetClientType(uiza.CategoryClientType)
```
Comment (titbk) [inline callback/client.go]: Đổi lại thành callbackClientType nha e

File: docs/CALLBACK.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+Example Request
+
+```golang
+import (
+    uiza "github.com/uizaio/api-wrapper-go"
+    "github.com/uizaio/api-wrapper-go/callback"
+)
+params := &uiza.CallBackIDParams{
+        ID:    uiza.String("Your ID")}
```
Comment (titbk) [inline docs/CALLBACK.md]: Param chỉ có một arg thì để một hàng lun e 
params := &uiza.CallBackIDParams{ID: uiza.String("Your ID")}

File: docs/CALLBACK.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+Example Request
+
+```golang
+import (
+    uiza "github.com/uizaio/api-wrapper-go"
+    "github.com/uizaio/api-wrapper-go/callback"
+)
+
+params := &uiza.CallBackIDParams{
+        ID:    uiza.String("Your ID")}
```
Comment (titbk) [inline docs/CALLBACK.md]: để thành 1 dòng nhé 
params := &uiza.CallBackIDParams{ID: uiza.String("Your ID")}

File: callback.go [hunk 4]
```diff
@@ -0,0 +1,61 @@
+package uiza
+
+type CallBackMethod string
+
+const (
+	CallBackMethodPOST CallBackMethod = "POST"
```
Comment (titbk) [inline callback.go]: Sao không dùng những cái có sẵn e 
http.MethodGet, ...

File: callback/client.go (hunk 1 above)
Comment (HanThiKimNgoc) [reply callback/client.go]: ok a

File: docs/CALLBACK.md (hunk 3 above)
Comment (HanThiKimNgoc) [reply docs/CALLBACK.md]: ok a

**Pull request merged**